tasks/schemas.py

```diff
@@ -0,0 +1,50 @@
+from ninja import Schema
+from typing import List, Dict
+from datetime import date, datetime
+from typing import Optional
+
+class TaskSchema(Schema):
+    id: int
+    user_email: str
+    task: str
+    due_by: datetime
+    priority: int
+    is_urgent: Optional[bool] = False
+
```

Comment:
If this schema should include all properties of a Task, you could build it based on the definition of the Task:

```suggestion
class TaskSchema(ModelSchema):
    class Meta:
        model = Task
        fields = "__all__"
```

Reply:
Smart I like it and updated it on next update